src/sentry/static/sentry/app/views/issueList/actions/reviewAction.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import ActionLink from 'app/components/actions/actionLink';
+import {IconIssues} from 'app/icons';
+import {t} from 'app/locale';
+import {Organization} from 'app/types';
+
+import {ConfirmAction, getConfirm, getLabel} from './utils';
+
+type Props = {
+  orgSlug: Organization['slug'];
+  onUpdate: (data?: any) => void;
+  primary?: boolean;
+  disabled?: boolean;
+  confirm?: ReturnType<typeof getConfirm>;
+  label?: ReturnType<typeof getLabel>;
+  onShouldConfirm?: (action: ConfirmAction) => boolean;
+};
+
+function ReviewAction({
+  disabled,
+  primary,
+  onShouldConfirm,
+  onUpdate,
+  confirm,
+  label,
+}: Props) {
+  return (
+    <ActionLink
+      type="button"
+      className={classNames('btn btn-sm action-merge', {
+        'btn-primary': primary,
+        'btn-default': !primary,
+      })}
+      data-test-id="button-acknowledge"
```

Comment:
we shouldn't use `data-test-id` for buttons, we have and should use `aria-label` to query